dojo/metrics/views.py

```diff
@@ -187,18 +187,19 @@ def finding_querys(prod_type, request):
                                                  test__engagement__product__prod_type__in=prod_type).prefetch_related(
             'test__engagement__product')
         # capture the accepted findings in period
-        accepted_findings = Finding.objects.filter(risk_acceptance__created__date__range=[start_date, end_date],
+        # set the start date of the month manually for acceepted_findings and accepted_findings_counts
+        accepted_findings = Finding.objects.filter(date__range=[start_date.replace(day=1), end_date], risk_accepted=True,
                                                    test__engagement__product__prod_type__in=prod_type). \
             prefetch_related('test__engagement__product')
-        accepted_findings_counts = Finding.objects.filter(risk_acceptance__created__date__range=[start_date, end_date],
+        accepted_findings_counts = Finding.objects.filter(date__range=[start_date.replace(day=1), end_date], risk_accepted=True,
```

Comment:
Does hard coding the start date affect the filters?

Reply:
Yep, for some reason, the original "start_date" variable is not the beginning of the month, so I add .replace(day=1) to make sure it starts from the first day of that month.

Reply:
So the following url will only find findings from 08/03/2021 and 08/04/2021 instead of 08/01/2021 and 08/04/2021?

/product/1/metrics?start_date=08/02/2021&end_date=08/03/2021

Reply:
It should not affect the product page metrics(product/1/metrics), this part of code is involving the overall metrics display (/metrics/product/type/1).
But I noticed another bugs on that page with the given url you have, and I'm trying to fix that right now.

Reply:
The same logic applies to both metric pages